### PR TITLE
Add a basic qchem function to pl-braket benchmark suite

### PR DESCRIPTION
**Pull request by soranjh**
This PR adds a qchem function to the pl-braket benchmark suite. The primitive circuit in this function will be replaced with a circuit built with excitation gates.

### CODE REVIEW

**Review by mariaschuld — APPROVED** [review]
Looks good...is this a benchmark case for a fast (i.e. actually runnable) circuit?

File: benchmarks/benchmark_functions/pl-braket-pipeline.py
Comment (albi3ro) [inline benchmarks/benchmark_functions/pl-braket-pipeline.py]: Should we define `n_wires` before the if-elif statements?  That value does not depend on the choice of device. Basically we get the same line duplicated multiple times.

File: benchmarks/benchmark_functions/pl-braket-pipeline.py
Comment (albi3ro) [inline benchmarks/benchmark_functions/pl-braket-pipeline.py]: Looking above, it looks like PennyLane Numpy is defined as `pnp` in this file, not `np`.  Maybe we should change that.

Also, if we will be using master PL instead of the latest release, we should change to using the updated random number generation method, see Line 72/73 of this file.

**Review by albi3ro — CHANGES_REQUESTED** [review]
Only change holding up approval would be making `np` match what gets imported.  That could be fixed by changing the import to `np` instead of `pnp` at the top.

**Review by ixfoduap — APPROVED** [review]
Looks good to me for a basic benchmark. Important to keep in mind that thanks to Olivia we now have decompositions for the excitation gates, so these could be used as well on all these devices.

Also, please address Christina's comments before merging.

**Comment by soranjh** [comment]
> Looks good...is this a benchmark case for a fast (i.e. actually runnable) circuit?

Yes, exactly. This is the most basic qchem circuit.

**Review by albi3ro — APPROVED** [review]
👍

**Pull request merged**
